.drone.yml

```diff
@@ -10,7 +10,13 @@ pipeline:
       - docker build -t "${DOCKER_IMAGE}-testing" -f Dockerfile-testing .
       - docker run -d --net host "${DOCKER_IMAGE}"
       - sleep 5
-      - docker run --net host "${DOCKER_IMAGE}-testing"
+      - printf "* lev-web unit and acceptence test logs *\n\n" > test.log
+      - tail -f test.log &
+      - TL=$!
+      - docker run --net host "${DOCKER_IMAGE}-testing" >> test.log 2>&1
+      - EC=$?
+      - kill "${TL}"
+      - exit "${EC}"
```

Comment:
Could this be simplified?
```bash
printf "* lev-web unit and acceptence test logs *\n\n" > test.log
docker run --net host "${DOCKER_IMAGE}-testing" 2>&1 | tee -a test.log
```

Reply:
Just checked and it looks like `tee` is present in the image. :-)
What do you think?